content/number-theory/Expo.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * Author: Brian Grana
+ * Date: 2024-10-02
+ * Source: own
+ * Description: Given two integers b and p, computes b
+ * raised to the power of p under modulo.
+ * Time: $O(log(p))$
+ * Status: tested on https://cses.fi/problemset/task/1095/
+ */
+#pragma once
+
+const long long mod = 1e9+7;
+long long expo(long long b, long long p) {
```

Comment:
assert nonnegative exponent? could avoid nasty bugs with the right shift

Reply:
I don't think this is necessary personally but I'll see what other people think

Reply:
did you see what other people think

Reply:
added comment // assert(p>0)